functions/src/index.ts

```diff
@@ -230,7 +231,35 @@ export const getCheckInData = functions.https.onRequest(async (req: functions.ht
   });
 })
 
+export const incrementReturnedStudentsCounter = functions.https.onRequest(async (req, res) => {
+  validateFirebaseIdToken(req, res);
+
+  await jwtClient.authorize();
+
+  const d = await sheets.spreadsheets.values.get({
+    auth: jwtClient,
+    spreadsheetId: SPREADSHEET_ID,
+    range: `${DASHBOARD_SHEET}!C16`
+  });
 
+  const v = d.data.values!;
+  const stu = v[0][0];
+  const new_stu = +stu + 1;
```

Comment:
This is the only comment that I wanted clarification on for comprehension: Is the addition of the "+" in front of stu, the casting to a number you were referencing?